### PR TITLE
feat(mobile): merge mobile repo into the backend repo

### LINKED ISSUE #9: Merge the mobile repo into the backend repo

### PR DESCRIPTION
**Pull request by denistakeda**
Closes #9

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,168 +1,17 @@
-# reactGo
+# Life Balance
 
-[![Dependency Status][dep-status-img]][dep-status-link] [![devDependency Status][dev-dep-status-img]][dev-dep-status-link]
-[![Gitter][gitter-img]][gitter-link] [![npm version][npm-badge]][npm-link]
+TODO: Add description
```
Comment (sv3k) [inline README.md]: Maybe we should state somewhere a note that this project is based on reactGo?

File: .gitignore [hunk 2]
```diff
@@ -16,6 +16,7 @@ logs
 pids
 *.pid
 *.seed
+android/.graddle
```
Comment (sv3k) [inline .gitignore]: Will this remove the bin files added in the previous commit?

Comment (denistakeda) [reply .gitignore]: This files are generated by react-native. So, no need to store it in git.

Comment (sv3k) [reply .gitignore]: I understand that, so will this change actually remove those unneeded bin files added in the previous commit?

Comment (denistakeda) [reply .gitignore]: Removed it from the git history also

File: README.md (hunk 1 above)
Comment (denistakeda) [reply README.md]: Done

**Comment by sv3k** [comment]
👍

**Pull request merged**